homeworks/hw7/bulls_and_cows/hw7_solution.py

```diff
@@ -0,0 +1,47 @@
+import random
+
+
+def generate_secret_number():
+
+    return ''.join(random.sample('0123456789', 4))
+
+
+def check_guess(secret, user_guess):
+
+    bulls_count = 0
+    cows_count = 0
+
+    for i in range(4):
+        if user_guess[i] == secret[i]:
+            bulls_count += 1
+
+    for x in user_guess:
+        if x in secret:
+            cows_count += 1
+
+    cows_count -= bulls_count
```

Comment:
You can avoid this `cows_count -= bulls_count`, if you will just use 1 for loop and if-elif inside
```suggestion
    for i in range(secret()):
        if user_guess[i] == secret[i]:
            bulls_count += 1
         elif user_guess[i] in secret[i]:
            cows_count += 1
```